components/brave_new_tab/resources/brave_new_tab_strings.grdp

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit-part>
+  <message name="IDS_NEW_TAB_BACKGROUND_SETTINGS_TITLE" desc="Title for background image settings">
+    Background Image
```

Comment:
Strings are deduped before translation, I believe. The idea behind adding everything as we go forward (rather than reusing existing resource IDs) is that we'll end up with the strings self-contained in this file, and can simply removed the unused ones in `brave_components_strings` when the transition is complete.